doc-content/enterprise-only/employee-rostering/wb-employee-rostering-example-deploying-in-central-proc.adoc

```diff
@@ -1,5 +1,5 @@
 [id='wb-employee-rostering-example-deploying-in-central-proc']
-= Deploy the employee rostering sample project in {CENTRAL}
+= Deploying the employee rostering sample project in {CENTRAL}
 
 {CENTRAL} includes a number of sample projects that you can use to get familiar with the product and its features. The employee rostering sample project has been designed and created to demonstrate the shift rostering use case for {PLANNER}. Use the following procedure to deploy and run the employee rostering sample in {CENTRAL}.
```

Comment:
"The employee rostering sample project is designed..." instead of "The employee rostering sample project has been designed.."